### PR TITLE
New activity bar Icons

### PR DESCRIPTION
**Pull request by SaraVieira**
This rebrands the UI feel of the activity bar with the new icons and spacing

### CODE REVIEW

**Comment by lbogdan** [comment]
Build for latest commit https://github.com/codesandbox/codesandbox-client/pull/3464/commits/8c27fc95462e1d184b682908afe3842bff2d7f26 is at https://pr3464.build.csb.dev/s/new.

File: packages/app/src/app/pages/Sandbox/Editor/Navigation/elements.ts [hunk 1]
```diff
@@ -14,66 +15,53 @@ export const Container = styled.div`
...
+    border: 0;
+    border-radius: ${props.theme.radii.small}px;
+    box-sizing: border-box;
+    padding: 0;
 
-  &:hover {
-    color: ${props => props.theme[`activityBar.foreground`] || css`white`};
-  }
+    &:hover {
+      background: ${props.theme.colors.sideBar.hoverBackground};
```
Comment (siddharthkp) [inline packages/app/src/app/pages/Sandbox/Editor/Navigation/elements.ts]: we should add `activityBar.hoverBackground` to polyfill

File: packages/components/src/utils/polyfill-theme.ts [hunk 2]
```diff
@@ -118,12 +119,18 @@ const polyfillTheme = vsCodeTheme => {
...
-    mutedForeground: withContrast(
-      uiColors.input.placeholderForeground,
-      uiColors.sideBar.background,
-      type
-    ),
+    mutedForeground,
+    activityBar: {
+      selected: uiColors.sideBar.foreground,
+      inactiveForeground: mutedForeground,
+    },
```
Comment (siddharthkp) [inline packages/components/src/utils/polyfill-theme.ts]: it's strange that the default border of the activity bar doesn't match the sidebar, probably a bug in the theme?

File: packages/app/src/app/pages/Sandbox/Editor/Navigation/elements.ts (hunk 1 above)
Comment (SaraVieira) [reply packages/app/src/app/pages/Sandbox/Editor/Navigation/elements.ts]: Done :)

File: packages/components/src/utils/polyfill-theme.ts (hunk 2 above)
Comment (SaraVieira) [reply packages/components/src/utils/polyfill-theme.ts]: naa, its normal as I see in other themes:

**Pull request merged**